locals.tf

```diff
@@ -1,23 +1,22 @@
 locals {
-  hostname                 = format("longhorn.apps.%s", var.base_domain)
-  hostname_withclustername = format("longhorn.apps.%s.%s", var.cluster_name, var.base_domain)
+  domain      = format("longhorn.apps.%s", var.base_domain)
+  domain_full = format("longhorn.apps.%s.%s", var.cluster_name, var.base_domain)
```

Comment:
@hbollon I used domain because we already had something similar on other modules, but I'm still not happy with `domain_full` :smile: